server/src/models/Dialog.js

```diff
@@ -0,0 +1,41 @@
+const Model = require("./Model")
+
+class Dialog extends Model {
+    static get tableName() {
+        return "dialogs"
+    }
+
+    static get jsonSchema() {
+        return {
+            type: "object",
+            required: ["userId"],
```

Comment:
this is ok, just know that by requiring the `userId` limits the types of Objection queries you will be able to use, such as `relate`